blocks/CONTROL_FLOW/LOGICAL_OPERATORS/AND/AND.py

```diff
@@ -0,0 +1,23 @@
+from flojoy import flojoy, Boolean
+
+
+@flojoy
+def AND(default: Boolean, a: Boolean) -> Boolean:
+    """The AND node takes two boolean data type and computs logical AND operation on them.
```

Comment:
Remove the first two works on all blocks and change `node` to `block` for all blocks